### ISSUE TITLE
A predictive model of gene expression using a deep learning framework

### ISSUE THREAD

**Issue by agitter** [greenelab/deep-review#210]
https://doi.org/10.1109/BIBM.2016.7822599

> With an unprecedented amount of data available, it is important to explore new methods for developing predictive models to mine this data for scientific discoveries. In this study, we propose a deep learning regression model based on MultiLayer Perceptron and Stacked Denoising Auto-encoder (MLP-SAE) to predict gene expression from genotypes of genetic variation. Specifically, we use a stacked denoising auto-encoder to train our regression model in order to extract useful features, and utilize the multilayer perceptron for backpropagation. We further improve our model by adding a dropout technique to prevent overfitting. Our results on a real genomic dataset show that our MLP-SAE model with dropout outperform Lasso, Random Forests, and MLP-SAE without dropout. Our study provides a new application of deep learning in mining genomics data, and demonstrates that deep learning has great potentials in building predictive models to help understand biological systems.

**Comment by gwaygenomics** [comment]
I couldn't find the full text for this paper, anyone else have any luck?

**Comment by agitter** [comment]
I get a "View Document" link directly under the title when I visit http://ieeexplore.ieee.org/document/7822599/.

A quick skim shows it to be a fairly standard model.  Input layer (features: SNP genotypes) -> autoencoder layer 1 -> autoencoder layer 2 -> output layer (regression).  They implement the model in pylearn2, but I don't see code.

They model yeast data from https://doi.org/10.1073/pnas.0408709102, which has 112 microarray samples.  "We extract the genotypes of 2,956 SNPs and gene expression profiles of 7,085 genes in the 112 samples", where I'm going to infer "genes" might mean "probes".   Performance is evaluated with mean squared error (MSE) in 5-fold cross validation.  They compare to L1-regularized linear regression and random forests.  I don't see strong evidence the neural network beats linear regression in any meaningful way.  The MSEs are almost the same under the best hyperparameters: 0.291 (LR) vs. 0.289 (NN).  Not to mention ~0.3 seems like a high MSE.

I'm closing this and will suggest we don't include it unless someone wants to convince me otherwise.
